docs/content/docs/connectors/table/hive/hive_read_write.md

```diff
@@ -409,6 +409,28 @@ This configuration is set in the `TableConfig` and will affect all sinks of the
   </tbody>
 </table>
 
+### Sink Parallelism
+
+The parallelism of writing data into Hive can be configured by the corresponding table option. By default, the parallelism is configured to being the same as the parallelism of its last upstream chained operator. When the parallelism which is different from the parallelism of the upstream parallelism is configured, the writer operator will apply the parallelism.
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+        <th class="text-left" style="width: 20%">Key</th>
+        <th class="text-left" style="width: 15%">Default</th>
+        <th class="text-left" style="width: 10%">Type</th>
+        <th class="text-left" style="width: 55%">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td><h5>sink.parallelism</h5></td>
```

Comment:
If we reuse configuration from `FileSystemOptions`, I prefer not to define this configuration in hive page, but add a link to FileSystem connector instead.

Reply:
Hi @lirui-apache, thx for ur suggest. Actually HiveSink in streaming mode is based on FileSystemSink, but seems to be not in batch mode. I'm not sure it is reasonable to link to FileSytemSink.  I also noticed the ref in `FileSystemOptions`:
> This class holds configuration constants used by filesystem(Including hive) connector. 

So I think it is Ok to just keep that. 

Well, correct me if I'm wrong~

Reply:
Hive batch sink also shares code with FileSystem connector, e.g. it also writes data using a `FileSystemOutputFormat`.

If `sink.parallelism` has the same meaning for both FS and hive connectors, it's better to avoid duplicated definitions. So that it makes things easier if we need to update the configuration in the future. That's also what we did in the [streaming write](https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/connectors/hive/hive_read_write.html#writing) section of the hive doc.

Reply:
I agree with @lirui-apache  here.

Reply:
@lirui-apache  Sure